file_processing.py

```diff
@@ -26,5 +26,5 @@ def read_file():
 
 def read_lines():
   with open(file_name) as f:
-    lines_array = f.readlines()
+    lines_array = f.readlines()  
```

Comment:
Šķiet, ka bez šī space var iztikt
```suggestion
    lines_array = f.readlines()
```